test/plugin/base_test.rb

```diff
@@ -274,6 +274,13 @@ def test_invalid_json_credentials
     end
   end
 
+  def test_unset_empty_credential_env_var
+    ENV[CREDENTIAL_PATH_VAR] = ''
+    setup_gce_metadata_stubs
+    create_driver
+    assert_nil ENV[CREDENTIAL_PATH_VAR]
+  end
```

Comment:
Can we add another test for when the environment variable is not present, just to ensure that we test both paths?

Reply:
+1.

Reply:
I thought about it. But that has been covered by a dozens of 'regular' tests that do not explicitly set this var. Because we delete the environment in `setup` for all tests:
```
  def setup
    Fluent::Test.setup
    # delete environment variables that googleauth uses to find credentials.
    ENV.delete(CREDENTIALS_PATH_ENV_VAR)
```

I feel It's kind of redundant to add another test. WDYT?

Reply:
I still like the idea of testing things in isolation. By having an explicit test for this, we don't create a passive dependency on other tests that happen to be doing the same thing by convenience.

Reply:
Added.